### PR TITLE
Technically untested but should work

### PR DESCRIPTION
**Pull request by dsaliba**

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -34,7 +36,8 @@
   
 
   private final DifferentialDriveTrain differentialDriveTrain = new DifferentialDriveTrain(Constants.leftDrivetrainIDs, Constants.rightDrivetrainIDs, Constants.driveTrainMotorType, Constants.driveMotorProfile);
-  private final ArcadeDrive arcadeDrive = new ArcadeDrive(Constants.driverStick, differentialDriveTrain);
+  private final ReversableArcadeDrive reversableArcadeDrive = new ReversableArcadeDrive(Constants.driverStick, differentialDriveTrain);
+  private final SwitchDriveDirrection switchDriveDirrection = new SwitchDriveDirrection(differentialDriveTrain);
```
Comment (InsightGit) [inline src/main/java/frc/robot/RobotContainer.java]: it would be nice if you could spell direction correctly because this is a major class name.

**Review by InsightGit — CHANGES_REQUESTED** [review]
Looks good just please correct the SwitchDriveDirrection class name to SwitchDriveDirection.
Normally I wouldn't be too picky but due to it being a major class name that will get referenced around the code, it would be better for our code style to do this. Thanks!

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (dsaliba) [reply src/main/java/frc/robot/RobotContainer.java]: fixed it

**Review by InsightGit — APPROVED** [review]
Looks good to merge now

**Pull request merged**